CHANGELOG.md

```diff
@@ -4,18 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [3.0.0] - ???
-
-TODO
+## [Unreleased]
 
 ### Added
  - VANISHED support in EXPUNGE responses and unsolicited responses (#172).
+ - SORT command extension (#178).
 
 ### Changed
  - MSRV increased to 1.43 for nom6 and bitvec
  - `expunge` and `uid_expunge` return `Result<Deleted>` instead of `Result<Vec<u32>>`.
-
-### Removed
+ - Idle `wait_keepalive_while` replaces `wait_keepalive` and takes a callback with an `UnsolicitedResponse` in parameter.
```

Comment:
@mordak I feel like we can be a bit more elaborate here — want to give a stab at updating this in a separate PR?